entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+TRACEE_EBPF_EXE=${TRACEE_EBPF_EXE:="/tracee/tracee-ebpf"}
+TRACEE_MOUNT_EXE=${TRACEE_MOUNT_EXE:="mount"}
+TRACEE_WEBHOOK_CONFIG=${TRACEE_WEBHOOK_CONFIG:="/tmp/tracee/integrations-config.yaml"}
```

Comment:
For my understanding: How is this used? I see Falco sidekick has a WebUI that I was expecting to see if I didn't pass in any arguments https://github.com/falcosecurity/falcosidekick-ui and https://github.com/falcosecurity/falcosidekick/blob/master/config_example.yaml#L164-L165

I'm just trying to configure Falco sidekick to be able to look at the events that Tracee will generate.

Reply:
IDK what's webUI, seems like a new feature added to master 2 days ago and yet unreleased

Reply:
I just configured it with MS Teams output to see that it works